docs/deep-link-schema.md

```diff
@@ -0,0 +1,88 @@
+# Deep link URL Schema
+
+Keep in mind that all the provided parameters should be URI encoded.
+
+## Connect to a wallet
+
+```bash
+https://wallet.superhero.com/address
+  ? x-success=<success-url>
+  & x-cancel=<cancel-url>
+```
+|URL Params|Description|
+|--|--|
+|`x-success` (required)| This is a callback URL in case user accept the connection attempt. Callback **must** contain: <br> - `{address}` parameter in order to get current address; <br> - `{networkId}` parameter in order to get current `networkId`.|
+|`x-cancel` (required)| This is a callback URL in case user have rejected connection attempt.|
+
+### Example
+
+```
+https://wallet.superhero.com/address?x-success=https%3A%2F%2Ftest.com%2Fsuccess-connection%3Faddress%3D%7Baddress%7D%26networkdId%3D%7BnetworkId%7D&x-cancel=https%3A%2F%2Ftest.com%2Ffail-connection
+```
+
+
+## Sign a transaction
+
+```bash
+https://wallet.superhero.com/sign-transaction
+  ? transaction=<Encoding.Transaction>
+  & networkId=<string>
+  & broadcast=<boolean>
+  & replace-caller=<boolean>
+  & x-success=<success-url>
+  & x-cancel=<cancel-url>
+```
+
+|URL Params|Description|
+|--|--|
+|`transaction` (required)|Valid transaction in [Encoding.Transaction](https://docs.aeternity.com/aepp-sdk-js/latest/api/enums/Encoding.html#Transaction) format `tx_`.|
+|`networkId` (required)|The `networkId` identifier for the network with which the wallet should sign your transaction.|
+|`broadcast` (optional)| This flag is for sending a signed transaction by the wallet.|
+|`replace-caller` (optional)| This flag is used to replace the caller's address of the `transaction` with the current address in the wallet.|
+|`x-success` (required)| This is a callback URL in case user have signed the transaction. If the `broadcast` flag is: <br> - `false/not set` callback **must** contain `{transaction}` parameter in order to get the signed transaction; <br> - `true` callback can have `{transaction-hash}` of the broadcasted transaction.|
```

Comment:
```suggestion
|`x-success` (required)| This is a callback URL in case user signs the transaction. If the `broadcast` flag is: <br> - `false/not set` callback **must** contain `{transaction}` parameter in order to get the signed transaction; <br> - `true` callback can have `{transaction-hash}` of the broadcasted transaction.|
```